### ISSUE TITLE
Cabbage widget accessibility issues

### ISSUE THREAD

**Issue by Jaseoffire** [rorywalsh/cabaiste#158]
I have found that at least two of the slider widgets do not have a way to control them via the computer keyboard. There does seem to be a number of accessibility issues with other widgets, like the text area, though I have not gotten around to fully testing them. I am on windows 11 using NVDA.

**Comment by rorywalsh** [comment]
The version of Juce used in Cabbage 2 does not have a very extensive range of accessibility features. They come with Juce v7. Cabbage 3 should be much better in this regard.
